commcare_connect/templates/opportunity/opportunity_detail.html

```diff
@@ -568,4 +570,23 @@ <h1 class="modal-title fs-5 text-white">{% translate "Import Catchment Areas" %}
   </div>
 </div>
 
+<div class="modal fade" id="resendInviteModal" tabindex="-1" aria-labelledby="resendInviteModalLabel" aria-hidden="true">
```

Comment:
I am happy to do that. I assume it would be shown at the top level, but the problem is that when the user scrolls down, they won't see the alert unless they manually scroll back up to the top, or we trigger the scroll.